x/gamm/types/constants.go

```diff
@@ -12,6 +12,8 @@ const (
 	// Raise 10 to the power of SigFigsExponent to determine number of significant figures.
 	// i.e. SigFigExponent = 8 is 10^8 which is 100000000. This gives 8 significant figures.
 	SigFigsExponent = 8
+	// TODO: Turn this into a param in the future.
```

Comment:
Great idea! Can we make an issue for this please and remove the comment?

Reply:
Created one! https://github.com/osmosis-labs/osmosis/issues/2058